Client/src/components/common/Link/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './index.css'
+import { Link } from 'react-router-dom';
+
```

Comment:
### For the import arrangement
- first: import third-party modules
- second: import your files 
and between them add a new line

Reply:
see code below